tests/NHSD.BuyingCatalogue.Testing.Data/Entities/SolutionEntity.cs

```diff
@@ -64,14 +91,15 @@ public sealed class SolutionEntity : EntityBase
                 @AboutUrl,
                 @ServiceLevelAgreement,
                 @WorkOfPlan,
+                @ImplementationDetail,
                 @LastUpdated,
                 @LastUpdatedBy
             );";
 
         public static async Task<IEnumerable<SolutionEntity>> FetchAllAsync()
         {
             const string selectSql = @"
-                SELECT s.Id,
+                SELECT s.Id as SolutionId,
```

Comment:
SQL keywords should be capitalized, so this line should be:

```sql
SELECT s.Id AS SolutionId
```

Reply:
Updated SQL keyword.